src/editors/playgroundController.ts

```diff
@@ -0,0 +1,125 @@
+import * as vscode from 'vscode';
+
+import ConnectionController from '../connectionController';
+import { ElectronRuntime } from '@mongosh/browser-runtime-electron';
+import { CompassServiceProvider } from '@mongosh/service-provider-server';
+import ActiveDBCodeLensProvider from './activeDBCodeLensProvider';
+import formatOutput from '../utils/formatOutput';
+import { createLogger } from '../logging';
+import { OutputChannel } from 'vscode';
+
+const log = createLogger('editors controller');
+
+/**
+ * This controller manages playground.
+ */
+export default class PlaygroundController {
+  _context?: vscode.ExtensionContext;
+  _connectionController?: ConnectionController;
+  _runtime?: ElectronRuntime;
+  _activeDB?: any;
+  _activeDBCodeLensProvider?: ActiveDBCodeLensProvider;
+  _outputChannel: OutputChannel;
+
+  constructor(context: vscode.ExtensionContext, connectionController: ConnectionController) {
+    this._context = context;
+    this._connectionController = connectionController;
+    this._outputChannel = vscode.window.createOutputChannel(
+      'Playground output'
+    );
+  }
+
+  createPlayground(): Promise<boolean> {
+    return new Promise((resolve, reject) => {
+      vscode.workspace
+        .openTextDocument({
+          language: 'mongodb',
+          content: '// The MongoDB playground'
+        })
+        .then((document) => {
+          vscode.window.showTextDocument(document);
+          resolve(true);
+        }, reject);
+    });
+  }
+
+  runDBHelpInPlayground(): Promise<boolean> {
+    return new Promise(async (resolve, reject) => {
+      let result;
+
+      try {
+        result = await this.evaluate('db.help()');
```

Comment:
When I run this I get the output:
<img width="711" alt="Screen Shot 2020-03-19 at 4 12 28 PM" src="https://user-images.githubusercontent.com/1791149/77082698-bce4ae80-69fc-11ea-8675-1d5b9473a42e.png">
I think we probably want to show output similar to how shell shows it:
<img width="737" alt="Screen Shot 2020-03-19 at 4 15 20 PM" src="https://user-images.githubusercontent.com/1791149/77082789-dede3100-69fc-11ea-8d42-95d2a9eb96c8.png">
Is this something that shell handles on its own end when displaying output?

Reply:
I think this API is not finished in mongosh yet. You can try to evaluate in the playground the regular `help`, it also does not return what shell does.